components/Signuser/Signuser.tsx

```diff
@@ -0,0 +1,83 @@
+import { useState } from "react";
+
+type AuthProps = {
+  type: string;
+};
+
+const AuthForm = ({ type }: AuthProps) => {
+  const [userState, setUserState] = useState({
+    email: "",
+    password: "",
+  });
+  const onSubmit = (e: any) => {
+    e.preventDefault();
+    console.log(userState);
+  };
+  return (
+    <div className="w-3/12 h-fit-content mx-auto mt-20">
+      <h1 className="text-center text-4xl font-semibold uppercase">{type}</h1>
+      <form onSubmit={(e) => onSubmit(e)} className="h-fit-content w-full mt-8">
```

Comment:
Refer material design on how to render forms and edit the design using those principles

Reply:
Sure, would make a prototype for that later.